challenge_12/active_record.rb

```diff
@@ -72,10 +54,8 @@
   elsif request.post? && request.path == '/create-post'
     puts 'Got a new POST request!'
 
-    blog = Blog.new(request.params)
-
-    # Write to database
-    database.execute('INSERT INTO blogs (title, content) VALUES (:title, :content)', blog.to_h)
+    # The request params can be passed directly to the Blog's create method - neat!
+    blog = Blog.create!(request.params)
```

Comment:
Ah, great point! Let me incorporate the use of strong params here ❤️

Reply:
That might be jumping the gun a little because we’ve not introduced Action Controller yet. So feel free to take a look, but off the top of my head I don’t know how easy it’ll be!

Reply:
(But Action Controller is coming up real soon now…)

Reply:
Right!! I was going to experiment with using strong parameters in this PR, but that would require transforming the request params into `ActionController::Parameters` and I suppose that would be jumping ahead a bit huh? 😂

Reply:
A little, and you’ll get that for free in the next challenge or two. So no harm trying, but don’t get bogged down in it is all.

Reply:
Hahaha I actually wrote my comment before seeing your next two comments (I guess we were reading each others minds 😉 ) - I'll hold off until the Action Controller challenges I think!